cli/bin/cherry.js

```diff
@@ -365,6 +368,22 @@ const buildSarifPayload = (projectName, branch, sha, occurrences) => {
   }
 }
 
+const buildSonarGenericImportPayload = (occurrences) => ({
+  issues: occurrences.map((occurrence) => ({
+    engineId: 'cherry',
+    ruleId: occurrence.metricName,
+    type: 'CODE_SMELL',
```

Comment:
I was looking for a list of options and found this. 
https://docs.sonarsource.com/sonarqube/latest/user-guide/issues/#issue-types

Apparently issue types are deprecated, but I don't really understand how it related to Clean Code, as they suggest.

Reply:
It still appears at code smell in the interface tho, and I can't find any Clean Code option for the generic issues format :/ wdyt about keep this for now?